component/notifications.libsonnet

```diff
@@ -0,0 +1,256 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+
+local inv = kap.inventory();
+local params = inv.parameters.openshift4_console;
+
+local namespace = {
+  metadata+: {
+    namespace: params.namespace,
+  },
+};
+
+local makeConsoleNotification(name, args) =
+  kube._Object('console.openshift.io/v1', 'ConsoleNotification', name) {
+    metadata+: {
+      labels+: {
+        'appuio.io/notification': 'true',
+      },
+    },
+    spec: {
+      text: args.text,
+      location: std.get(args, 'location', 'BannerTop'),
+      color: std.get(args, 'color', '#fff'),
+      backgroundColor: std.get(args, 'backgroundColor', '#2596be'),
+      link: std.get(args, 'link'),
```

Comment:
What happens if `link` is `null`, is the notification controller/operator smart enough to ignore fields with value `null` or should we `std.prune()` the spec here?

Reply:
I'll check

Reply:
`std.prune` is necessary, otherwise the consolenotification is rejected as invalid